lib/validator.rb

```diff
@@ -22,6 +22,14 @@ def valid_field?(field, value)
     }.fetch(field, true)
   end
 
+  def valid_index?(value, max_index)
+    value.to_i < max_index
+  end
+
+  def valid_field_name?(value)
+    Constants::FIELDS_TO_DISPLAY_NAMES.value?(value.capitalize + ': ')
```

Comment:
:thinking: why do you do this?

* what if you change the structure of the display name and it is not anymore the field capitalized with a colon?
* what if you change the language and you don't have **phone/Phone** anymore, but rather **phone/Número de teléfono**? This will fail.

The keys that are useful for code don't always translate well to be useful in human language.
Symbols is what we use in code, as a representation of a concept, we don't use strings that we print to the screen, which can be in any format or language.

All you need for the field name to be valid is to be one of the supported fields. 
So you can just check that it is in the array of keys.

```ruby
  def valid_field_name?(field_name)
    Constants::FIELDS_TO_DISPLAY_NAMES.keys.include?(field_name.downcase.to_sym)
  end
```